### DISCUSSION TITLE
Delete KV Database

### DISCUSSION THREAD

**Discussion by johnpolacek** [vercel/community#4292] [Help] [answered]
### Summary

I must be missing something, but There does not seem to be any way in the UI to remove or delete a database after you create it?

### Steps to Reproduce

Create a KV database (hobby plan). Where do I go to delete or remove it from my account.

**Comment by amyegan** [comment, marked as answer]
Hi @johnpolacek! You can delete a KV database if you go to your account dashboard homepage, then the `Storage` tab, then click to open details page for the database, go to the `Settings` for the database, and scroll to the bottom to find the `Delete Database` section. 

If the database is currently connected to a project, you will see the message "You must remove all connected projects before deleting this database." You can remove the connection from the `Projects` tab. After any project connections are removed, you will be able to go back to the `Settings` tab and delete the database.
